src/middleware/api.js

```diff
@@ -15,6 +20,39 @@ import {
 
 let socket = null;
 
+/**
+ *
+ * Add configured expanders to an api call for an action
+ * @function addExpandersToPath
+ * @param {string} path The url/path including the querystring
+ * @param {*} type The action type
+ * @returns {string} The url/path with the configured expanders added to the query string
+ */
+export function addExpandersToPath(path, type) {
+  const { settings } = config;
+  const { apiExpanders = [] } = settings;
+  if (!path) {
+    return path;
+  }
+  const pathPart = path.split('?')[0] || '';
+  const Expanders = apiExpanders
+    .map((reg) => {
+      const match = matchPath(pathPart, reg.match);
+      return match ? reg[GET_CONTENT] : null;
```

Comment:
We need to filter this for all possible actions that could come in the config, not only `GET_CONTENT`, pass it as a parameter, then filter it too.

Also, I think this overall filter can be improved and simplified a lot.

Reply:
Done!